src/opencensus.erl

```diff
@@ -195,5 +194,4 @@ generate_trace_id() ->
 %%--------------------------------------------------------------------
 -spec generate_span_id() -> non_neg_integer().
 generate_span_id() ->
-    <<Id:64>> = crypto:strong_rand_bytes(8),
-    Id.
+    rand:uniform(18446744073709551616).
```

Comment:
On pre 20 we might still be short 6 bits here in the worst case, not sure worth the effort for making a special function OTOH who knows?